### PR TITLE
Actor which will inform users about differences in Python

### PR DESCRIPTION
**Pull request by frenzymadness**
I think that the subject says it all. I'll add a link to KB article into the message asap.

Cc @encukou @Dormouse759

### CODE REVIEW

**Comment by frenzymadness** [comment]
Cc @hhorak

File: repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from leapp.actors import Actor
+from leapp.libraries.common import reporting
+from leapp.tags import ChecksPhaseTag
+from leapp.models import Report
+
+message = """
+Warning: In RHEL 8, there is no 'python' command. Python 3 is the primary Python version and Python 2 is available with limited support and limited set of packages.
+Read more here: FIXME
```
Comment (encukou) [inline repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py]:
```suggestion
Read more here: https://access.redhat.com/articles/4055961
```

Sadly, the article is not accessible to the public, nor (yet) to customers. It reuses information from the [Python in RHEL 8 blog post](https://developers.redhat.com/blog/2018/11/14/python-in-rhel-8/) and [RHEL 8 beta release notes](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html-single/configuring_basic_system_settings/index#using-python3).

**Comment by frenzymadness** [comment]
I'll add one more report message with low severity about incompatibilities between Python 2 and 3 and with a direct link to the conservative porting guide.

**Comment by frenzymadness** [comment]
Because the KB article summarises it very well, I decided to add just a small note about backwards incompatibility instead of a new report message with low severity. Now, it's all in one with high severity so it should be read by everyone.

PR is ready for review/squash/merge

File: repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from leapp.actors import Actor
+from leapp.libraries.common import reporting
+from leapp.tags import ChecksPhaseTag
+from leapp.models import Report
+
+message = """
+Warning: In RHEL 8, there is no 'python' command. Python 3 (backward incompatible) is the primary Python version and Python 2 is available with limited support and limited set of packages.
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py]:
```suggestion
In RHEL 8, there is no 'python' command. Python 3 (backward incompatible) is the primary Python version and Python 2 is available with limited support and limited set of packages.
```

Comment (pirat89) [reply repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py]: Anyway, as we discussed, it's possible to setup `alternatives` so you should ideally generate remediation instruction. See the function `report_with_remediation` where you would put info that user would setup python2 or python3 to be ablle to use the `python` command. I think this ismessage you would provide to the customer.

**Comment by pirat89** [comment]
@frenzymadness I sent you PR with proposals how the actor would looks like https://github.com/frenzymadness/leapp-repository/pull/1

It's just suggestion. You can change it as you wish.

File: repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py (hunk 2 above)
Comment (frenzymadness) [reply repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py]: PR in my fork merged

File: repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+class PythonInformUser(Actor):
+    name = "python_inform_user"
+    description = "This actor informs the user of differences in Python version and support in RHEL 8."
+    consumes = ()
+    produces = (Report,)
+    tags = (ChecksPhaseTag,)
+
+    def process(self):
+        title = "Difference in Python versions and support in RHEL 8"
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py]:
```suggestion
        url = "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/configuring_basic_system_settings/#using-python3" # noqa E501
        title = "Difference in Python versions and support in RHEL 8"
```

File: repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        severity = "high"
+        summary = ("In RHEL 8, there is no 'python' command."
+                   " Python 3 (backward incompatible) is the primary Python version"
+                   " and Python 2 is available with limited support and limited set of packages."
+                   " Read more here: https://access.redhat.com/articles/4055961")
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/pythoninformuser/actor.py]:
```suggestion
                   " Read more here: {}".format(url))
```

**Pull request merged**